### PR TITLE
Add snapcraft.yaml (support snap packaging)

### PR DESCRIPTION
**Pull request by davecore82**
I thought I'd add a snapcraft.yaml file to support building a snap for nudoku. Once this file is added, someone will still need to add this snap to the snap store[1]. 

[1] https://docs.snapcraft.io/c-c-applications/7817

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: nudoku
+version: master
```
Comment (jubalh) [inline snap/snapcraft.yaml:2]: I'm not familiar with snap. But why does the version here say `master` and below the 1.0.0 tarball is used? Is `master` here something different than git master?

**Review by jubalh — COMMENTED** [review]
Thanks for your contribution. Just one thing I'm not clear about.

**Comment by jubalh** [comment]
> Once this file is added, someone will still need to add this snap to the snap store.

While I appreciate this patch and will certainly merge it, I'm not interested in snap myself. Could you add it to the snap store once this is merged here?

**Comment by davecore82** [comment]
Sure I can take care of adding the snap to the store.

For the comment about the version, I wasn't too sure what to put there. Reading https://docs.snapcraft.io/snapcraft-yaml-reference/4276 , it looks like maybe it should be git ("will be replaced by a git describe based version string").

I can take care of those details when it's time to snap it for real.

**Comment by jubalh** [comment]
@davecore82 still any interest in this PR?

**Comment by davecore82** [comment]
@jubalh Thanks for reaching out, I'll close it.

**Pull request closed without merging**